langs/c-sharp/c-sharp.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.11.0"/>
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.12.0-3.final"/>
```

Comment:
I think in the dotnet world, `.final` is a bit like `rc` and we should wait for the non-suffixed version to appear.